Cohort2/team4/server/src/config/connection.ts

```diff
@@ -0,0 +1,22 @@
+const Sequelize = require('sequelize');
+require('dotenv').config();
+
+const sequelize = new Sequelize(
+  process.env.DB_NAME,
+  process.env.DB_USER || 'ix1v91q5n3s6myphw3hs',
+  process.env.DB_PASSWORD || 'pscale_pw_DZW0GlTvKGk9AdOofRgs85ijXXGuqTgCN1FCHmSFIDJ',
```

Comment:
This definitely should not be here. If the password is not set, this needs to fail / abort.